tests/atlas-tests.hs

```diff
@@ -30,6 +30,7 @@ import           GeniusYield.Types
 main :: IO ()
 main = do
     configs <- forM ["maestro-config.json", "blockfrost-config.json"] coreConfigIO
+    (providerToken, netId) <- findMaestroTokenAndNetId configs
```

Comment:
Since we have configs for both Maestro & Blockfrost. Could you bring this test for Blockfrost as well in another PR? `providerTests` already received `configs` & thus inside `providerTests`, you can get keys & network id as well.